auth_mellon_util.c

```diff
@@ -116,6 +116,10 @@ int am_validate_redirect_url(request_rec *r, const char *url)
 
     /* Sanity check of the scheme of the domain. We only allow http and https. */
     if (uri.scheme) {
+	/* http and https schemes without hostname are invalid. */
+        if (!uri.hostname) {
+            return HTTP_BAD_REQUEST; 
+    	}
```

Comment:
Could you fix the indentation of the comment and closing curly bracket?

Reply:
Hi @olavmrk. Sure, it should be fixed now.